### DISCUSSION TITLE
7 Hour Polling Issue

### DISCUSSION THREAD

**Discussion by kmercer6** [zwave-js/zwave-js#6340] [General]
Not sure if it was a Home Assistant 2023.9 or a new zwave-js issue.  Roughly September 4th my Zwave devices that are connected to a HUSBZB-1 which provides Zigbee (works fine) and Zwave started experiencing issues.  I utilize 2 Aeotec Devices (Home Energy Monitor and Leak Sensor) that started giving me results every roughly 7 hours versus 30 seconds they are configured for I do no see any drops and response times show fast when it works.  After updates by both HA and Zwave the issue isn't resolved sadly.

Is this a known bug or what logs might be helpful in correcting this issue?  I have seen atleast 1 other user on the HA forums with this issue with no resolution.

**Comment by AlCalzone** [comment]
Please make a [driver log](https://zwave-js.github.io/zwave-js-ui/#/troubleshooting/generating-logs?id=driver-logs), loglevel `debug` and attach it here as a file (drag & drop into the text field). Using Z-Wave JS UI is easiest because you can just log to file and leave it running.

In HA you can [do this](https://www.home-assistant.io/integrations/zwave_js#how-to-access-the-z-wave-logs), but you have to keep the log window open.

**Comment by kmercer6** [comment]
[zwavejs_current.log](https://github.com/zwave-js/node-zwave-js/files/12870141/zwavejs_current.log)

**Reply by AlCalzone** [reply]
The updates you are seeing are triggered by Z-Wave JS requesting the meter values because it hasn't received any within the last 6 hours (roughly). The time delta of roughly 21000 seconds since the last measurement advertised by the reports also confirms that the device hasn't sent any report for 6 hours at this point.

Possible reasons could be:
1. The devices aren't actually set up to report every 30s. Some require you to also select **what** to report. Side note: time-based reports are rarely a good idea --> https://zwave-js.github.io/node-zwave-js/#/troubleshooting/network-health?id=optimizing-the-reporting-configuration
2. The devices are missing a lifeline association (where to report to). This could be because of an error during the initial interview and **may** be fixed by a re-interview. If not, you can also use Z-Wave JS UI to manually set an association.

**Comment by kmercer6** [comment]
@AlCalzone if there are any other logs please let me know.  

As an update to this I switched out the Aeotec Z-Stick 7 (700 Series Firmware 7.17.2) and running Home Assistant 2023.10.1
